src/ZRMain.py

```diff
@@ -87,7 +87,13 @@ def run(self):
         self.step_index += 1
 
     def jump(self):
-        pass
+        self.image = self.jump_img

```

Comment:
jump_img is a list, the program probably craches since it regards the list as an image. Try accessing the first index of the list (self.jump_img[0])